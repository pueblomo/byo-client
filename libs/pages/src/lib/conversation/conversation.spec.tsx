import React from 'react';
import { render } from '@testing-library/react-native';

import Conversation from './conversation';

describe('Conversation', () => {
  it('should render successfully', async () => {
    const tree = render(<Conversation />);
    await new Promise((f) => setTimeout(f, 1000));
    expect(tree).toMatchSnapshot();
  });
});
