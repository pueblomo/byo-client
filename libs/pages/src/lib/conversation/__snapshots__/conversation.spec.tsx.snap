// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Conversation should render successfully 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "justifyContent": "space-between",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#ffffff",
        "borderBottomColor": "#f2f2f2",
        "borderBottomWidth": 0.5,
        "flexDirection": "row",
        "justifyContent": "space-between",
        "paddingHorizontal": 10,
        "paddingVertical": 10,
      }
    }
    testID="headerContainer"
  >
    <RNCSafeAreaView
      edges={
        Array [
          "left",
          "top",
          "right",
        ]
      }
      style={
        Object {
          "flexDirection": "row",
          "width": "100%",
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "flex-start",
            "flex": 1,
          }
        }
      />
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 3,
          }
        }
      >
        <Text
          numberOfLines={1}
          text="Peter Schwanz"
        >
          Peter Schwanz
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "flex-end",
            "flex": 1,
          }
        }
      />
    </RNCSafeAreaView>
  </View>
  <BVLinearGradient
    colors={
      Array [
        4294573055,
        4294573055,
      ]
    }
    endPoint={
      Object {
        "x": 0.5,
        "y": 1,
      }
    }
    locations={null}
    startPoint={
      Object {
        "x": 0.5,
        "y": 0,
      }
    }
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <RCTScrollView
      onContentSizeChange={[Function]}
    >
      <View>
        <View
          style={
            Object {
              "flex": 1,
            }
          }
        >
          <View
            style={
              Object {
                "shadowColor": "#171717",
                "shadowOffset": Object {
                  "height": 3,
                  "width": 0,
                },
                "shadowOpacity": 0.4,
                "shadowRadius": 2,
              }
            }
          >
            <View
              collapsable={false}
              style={
                Object {
                  "alignSelf": "flex-start",
                  "backgroundColor": "#969696",
                  "borderRadius": 10,
                  "marginBottom": 10,
                  "marginLeft": 20,
                  "marginRight": 20,
                  "marginTop": 10,
                  "maxWidth": "60%",
                  "padding": 10,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                  ],
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "fontWeight": "bold",
                    },
                    Object {
                      "color": "#F9FBFF",
                    },
                  ]
                }
              >
                Hamena
              </Text>
            </View>
          </View>
          <View
            style={
              Object {
                "shadowColor": "#171717",
                "shadowOffset": Object {
                  "height": 3,
                  "width": 0,
                },
                "shadowOpacity": 0.4,
                "shadowRadius": 2,
              }
            }
          >
            <View
              collapsable={false}
              style={
                Object {
                  "alignSelf": "flex-start",
                  "backgroundColor": "#969696",
                  "borderRadius": 10,
                  "marginBottom": 10,
                  "marginLeft": 20,
                  "marginRight": 20,
                  "marginTop": 10,
                  "maxWidth": "60%",
                  "padding": 10,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                  ],
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "fontWeight": "bold",
                    },
                    Object {
                      "color": "#F9FBFF",
                    },
                  ]
                }
              >
                Hamena
              </Text>
            </View>
          </View>
          <View
            style={
              Object {
                "shadowColor": "#171717",
                "shadowOffset": Object {
                  "height": 3,
                  "width": 0,
                },
                "shadowOpacity": 0.4,
                "shadowRadius": 2,
              }
            }
          >
            <View
              collapsable={false}
              style={
                Object {
                  "alignSelf": "flex-start",
                  "backgroundColor": "#969696",
                  "borderRadius": 10,
                  "marginBottom": 10,
                  "marginLeft": 20,
                  "marginRight": 20,
                  "marginTop": 10,
                  "maxWidth": "60%",
                  "padding": 10,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                  ],
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "fontWeight": "bold",
                    },
                    Object {
                      "color": "#F9FBFF",
                    },
                  ]
                }
              >
                lkajsdlkjask alsdkjalkjd alsdkjlka dalkj
              </Text>
            </View>
          </View>
          <View
            style={
              Object {
                "shadowColor": "#171717",
                "shadowOffset": Object {
                  "height": 3,
                  "width": 0,
                },
                "shadowOpacity": 0.4,
                "shadowRadius": 2,
              }
            }
          >
            <View
              collapsable={false}
              style={
                Object {
                  "alignSelf": "flex-start",
                  "backgroundColor": "#969696",
                  "borderRadius": 10,
                  "marginBottom": 10,
                  "marginLeft": 20,
                  "marginRight": 20,
                  "marginTop": 10,
                  "maxWidth": "60%",
                  "padding": 10,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                  ],
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "fontWeight": "bold",
                    },
                    Object {
                      "color": "#F9FBFF",
                    },
                  ]
                }
              >
                Hamena
              </Text>
            </View>
          </View>
          <View
            style={
              Object {
                "shadowColor": "#171717",
                "shadowOffset": Object {
                  "height": 3,
                  "width": 0,
                },
                "shadowOpacity": 0.4,
                "shadowRadius": 2,
              }
            }
          >
            <View
              collapsable={false}
              style={
                Object {
                  "alignSelf": "flex-start",
                  "backgroundColor": "#969696",
                  "borderRadius": 10,
                  "marginBottom": 10,
                  "marginLeft": 20,
                  "marginRight": 20,
                  "marginTop": 10,
                  "maxWidth": "60%",
                  "padding": 10,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                  ],
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "fontWeight": "bold",
                    },
                    Object {
                      "color": "#F9FBFF",
                    },
                  ]
                }
              >
                Hamena
              </Text>
            </View>
          </View>
          <View
            style={
              Object {
                "shadowColor": "#171717",
                "shadowOffset": Object {
                  "height": 3,
                  "width": 0,
                },
                "shadowOpacity": 0.4,
                "shadowRadius": 2,
              }
            }
          >
            <View
              collapsable={false}
              style={
                Object {
                  "alignSelf": "flex-end",
                  "backgroundColor": "#79C7C5",
                  "borderRadius": 10,
                  "marginBottom": 10,
                  "marginLeft": 20,
                  "marginRight": 20,
                  "marginTop": 10,
                  "maxWidth": "60%",
                  "padding": 10,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                  ],
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "fontWeight": "bold",
                    },
                    Object {
                      "color": "#F9FBFF",
                    },
                  ]
                }
              >
                Nerv nicht
              </Text>
            </View>
          </View>
        </View>
      </View>
    </RCTScrollView>
  </BVLinearGradient>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#ffffff",
        "flexDirection": "row",
        "height": "8%",
        "justifyContent": "space-around",
      }
    }
  >
    <TextInput
      onChangeText={[Function]}
      placeholder="Type your message"
      placeholderTextColor="#79C7C5"
      style={
        Object {
          "color": "#79C7C5",
          "width": "80%",
        }
      }
      value=""
    />
    <View
      style={
        Array [
          Object {
            "overflow": "hidden",
          },
          Object {
            "borderRadius": 2,
          },
          Object {
            "marginRight": 10,
            "width": 50,
          },
          false,
        ]
      }
      testID="RNE_BUTTON_WRAPPER"
    >
      <View
        accessibilityRole="button"
        accessibilityState={
          Object {
            "busy": false,
            "disabled": false,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#2089dc",
              "borderColor": "#2089dc",
              "borderRadius": 30,
              "borderWidth": 0,
              "flexDirection": "row",
              "justifyContent": "center",
              "padding": 5,
              "paddingHorizontal": 10,
            }
          }
        >
          <Text
            allowFontScaling={false}
            selectable={false}
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontSize": 12,
                },
                undefined,
                Object {
                  "fontFamily": "Feather",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
